styles/global-styles.scss

```diff
@@ -1,3 +1,4 @@
+@import "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css";
```

Comment:
How about using library instead? https://fontawesome.com/v5.15/how-to-use/on-the-web/using-with/react

Only those icons will be added to the final bundle which we'll use, right? Reducing the final bundle size compared to having to add whole of the css file from CDN to our bundle and then use only 1/2 out of it?

Please drop in your thoughts :)

Reply:
I initially thought of using this library, but for one icon I thought let's use it this way.

I am thinking of let's implement our own icons and use across RDS. Design team can help us.

Coming to increase in bundle size, don't worry about that.. next js has the feature of tree shaking, so in thr production build all the unused classes will be removed.